### PR TITLE
Enabled reading config files for vsphere e2e tests.

### PR DESCRIPTION
**Pull request by huffmanca**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

Enables the ability to pass vsphere configuration files for e2e tests outside of an environment variable.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Review by timothysc — APPROVED** [review]
/approve 

But it looks like tests are failing.

**Comment by ttousai** [comment]
Bug triage for 1.17 here with a gentle reminder that code freeze for this release is on Nov. 18. Is this PR still intended for 1.17?

**Comment by ttousai** [comment]
***Correction:*** Code freeze is on Thursday, November 14.

Sorry for the mix up.

File: build/root/BUILD.root [hunk 1]
```diff
@@ -89,6 +89,9 @@ filegroup(
         "//cmd:all-srcs",
         "//docs:all-srcs",
         "//hack:all-srcs",
+        "//kubernetes/cluster:all-srcs",
+        "//kubernetes/docs:all-srcs",
```
Comment (andrewsykim) [inline build/root/BUILD.root]: Why is this here?

**Comment by josiahbjorgaard** [comment]
@alejandrox1 Will this be making it into v1.17 before code freeze?

**Comment by akutz** [comment]
/hold 

In case others want to provide feedback

**Review by alejandrox1 — COMMENTED** [review]
This looks good and tests are passing.
Thank you for picking this up @huffmanca 
/lgtm
